met-api/src/met_api/config.py

```diff
@@ -82,6 +106,17 @@ class _Config():  # pylint: disable=too-few-public-methods
     S3_REGION = os.getenv('S3_REGION')
     S3_SERVICE = os.getenv('S3_SERVICE')
 
+    S3_CONFIG = {
+        'DEFAULT': {
+            'S3_BUCKET': os.getenv('S3_BUCKET'),
+            'S3_ACCESS_KEY_ID': os.getenv('S3_ACCESS_KEY_ID'),
+            'S3_SECRET_ACCESS_KEY': os.getenv('S3_SECRET_ACCESS_KEY'),
+            'S3_HOST': os.getenv('S3_HOST'),
+            'S3_REGION': os.getenv('S3_REGION'),
+            'S3_SERVICE': os.getenv('S3_SERVICE')
+        }
+    }
+
```

Comment:
can we please add a section of GDX or some tenant so that there is an example in the config?

Reply:
We had a discussion, where you confirmed we will not do this